scripts/itkdev-docker-compose

```diff
@@ -16,6 +16,88 @@ function find_script_dir {
 script_dir=$(find_script_dir)
 script_path=$script_dir/$(basename "${BASH_SOURCE[0]}")
 
+function template_install {
+  local force=0
+  local list=0
+  local name=""
+
+  for var in "$@"
+  do
+    case "$var" in
+      --force)
+        force=1
+        ;;
+      --list)
+        list=1
+        ;;
+      *)
+        name="$var"
+        ;;
+    esac
+  done
+
+  local templates_dir=$script_dir/../templates
+  local source_dir=$templates_dir/$name/
+  local target_dir=$PWD
```

Comment:
Will this work if templates exists and user stands in `site/all/`? You could use the check for docker-compose backwards search function or a warning that you should be in the "main" folder when installing templates.

Reply:
We don't have a `docker-compose.yaml` yet (since we're about to install it), so we cannot search backwards for it.

A “Are you sure you want to install into …” confirmation question will make sense – I'll add it.

Reply:
Fixed